libmscore/chordlist.cpp

```diff
@@ -295,16 +295,25 @@ static void readRenderList(QString val, QList<RenderAction>& renderList)
       {
       renderList.clear();
       QStringList sl = val.split(" ", QString::SkipEmptyParts);
-      foreach(const QString& s, sl) {
+      for (const QString& s : sl) {
             if (s.startsWith("m:")) {
                   QStringList ssl = s.split(":", QString::SkipEmptyParts);
                   if (ssl.size() == 3) {
+                        // m:x:y
                         RenderAction a;
                         a.type = RenderAction::RenderActionType::MOVE;
                         a.movex = ssl[1].toDouble();
                         a.movey = ssl[2].toDouble();
                         renderList.append(a);
                         }
+#if 0
```

Comment:
Why?

Reply:
As per the TODO comment, I'd like to eventually add support for something like m:superscript that would allow a chord description file to explicitly say which symbols should get superscripted, but have the amount of superscripting come from the new style setting rather than be hard-coded into the XML file.